gui/wxpython/mapswipe/frame.py

```diff
@@ -408,6 +408,16 @@ def OnApplyInputChanges(self):
                         'second']
                     GError(parent=self, message=message)
                     return
+            elif not (res1 and res2) and not (first and second):
+                if not (first or second):
+                    GError(
+                        parent=self,
+                        message=_(
+                            'Choose top/left and bottom/right '
+                            'raster map, please.'
+                        ),
+                    )
+                    return
```

Comment:
I consider empty inputs as valid in this case. It's useless, but there is nothing wrong with not showing any data in one or both windows. So I would not prevent user from hitting OK when they don't provide any input.

Reply:
All right, I've removed the error dialog.